### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by lilarin**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    results_cache = {}
+
+    def inner(*args) -> dict:
```
Comment (DevSpaciX) [inline app/main.py]: Add here type hint for args please :) And you not  returning dict , it can be any type ( int , float etc. )

**Review by DevSpaciX — CHANGES_REQUESTED** [review]
Add / change type hints pls :)

File: app/main.py (hunk 1 above)
Comment (lilarin) [reply app/main.py:7]: > Add here type hint for args please :) And you not returning dict , it can be any type ( int , float etc. )

Comment (lilarin) [reply app/main.py]: Yes, of course, but I was just testing under tests and wanted to get rid of "Any" to lighten the code while keeping functionality

**Review by DevSpaciX — APPROVED** [review]
Good job!

**Pull request closed without merging**